app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -159,6 +159,13 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
                 // Cache focused DOM element just before WebView gains focus. See comment in
                 // FocusedDOMElementCacheInterface for details
                 engineView.focusedDOMElement.cache()
+
+                // EngineView focus may be lost after waking up from sleep & screen saver.
+                // Forcibly request focus onStart(), after DOMElement cache, IFF webRenderFragment
```

Comment:
Is `IFF` a typo for `if`, or an acronym I'm missing?

Reply:
if and only if